.github/workflows/1_create_release_pr.yml

```
@@ -17,38 +17,42 @@ jobs:
     timeout-minutes: 10
     steps:
 
-    - name: Sanitise workflow inputs # Should be 1st step
-      uses: cylc/release-actions/stage-1/sanitize-inputs@v1
-
-    - name: Checkout repo
-      uses: actions/checkout@v2
-      with:
-        ref: ${{ env.BASE_REF }}
-
-    - name: Setup Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.x'
-
-    - name: Create & checkout PR branch
-      uses: cylc/release-actions/stage-1/checkout-pr-branch@v1
-
-    - name: Set the package version
-      uses: cylc/release-actions/stage-1/set-python-package-version@v1
-      with:
-        init-file: 'cylc/flow/__init__.py'
-        pypi-package-name: 'cylc-flow'
-
-    - name: Update "released on" date in changelog
-      continue-on-error: true
-      uses: cylc/release-actions/stage-1/update-changelog-release-date@v1
-      with:
-        changelog-file: 'CHANGES.md'
-
-    - name: Test build
-      uses: cylc/release-actions/build-python-package@v1
-
-    - name: Create pull request
-      uses: cylc/release-actions/stage-1/create-release-pr@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Sanitise workflow inputs # Should be 1st step
+        uses: cylc/release-actions/stage-1/sanitize-inputs@v1
+
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ env.BASE_REF }}
+          fetch-depth: 0  # need to fetch all commits to check contributors
+
+      - name: Check CONTRIBUTING.md
+        uses: cylc/release-actions/check-shortlog@c8205dcfb0df1661b99e2c240b75787c4fa02aa3
```

```suggestion
        uses: cylc/release-actions/check-shortlog@v1
```
I'll bump the v1 tag on there

---

Happy using SHA, if we are going to bump the tag might as well create one called "latest".

---

The idea is we keep bumping v1 (actually it's a branch not a tag) until we introduce a backwards incompatible change, then we start v2 so we don't break existing workflows across the repos